bin/run.sh

```diff
@@ -31,50 +33,58 @@ mkdir -p "${output_dir}"
 
 # Run the tests for the provided implementation file
 cd "$solution_dir" || exit
+
 # Output is saved at this location
 echo "==== ${slug}: testing..."
+
 test_output=""
-if ! [ -s *.bal ]; then
```

Comment:
Also, this was an error waiting to happen: if that wildcard expanded to 2 files, the warning would result.